defaults_test.go

```diff
@@ -226,8 +229,36 @@ func TestMustSet(t *testing.T) {
 
 }
 
+func TestOverrideDefaultTrueBoolToFalse(t *testing.T) {
+	type example struct {
+		BoolTrue        bool  `default:"true"`
+		BoolPointerTrue *bool `default:"true"`
```

Comment:
This is the workaround to have a field that defaults to `true` that a user can still set to `false` and not have defaults.Set() change it back to `true`.